README.md

```diff
@@ -1,8 +1,8 @@
-# Profiles
+# GP Finder
 
-[![Coverage Status](https://coveralls.io/repos/github/nhsuk/profiles/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/profiles?branch=master)
+[![Coverage Status](https://coveralls.io/repos/github/nhsuk/gp-finder/badge.svg?branch=master)](https://coveralls.io/github/nhsuk/gp-finder?branch=master)
 
-A collection of health service provider profiles. Initially containing GP practices and Pharmacies.
+A GP finder application. Will used for booking a GP and to find a new GP to register with.
 
 
 ## Environment variables
```

Comment:
Add `DOCKER_COMPOSE_VERSION` as it is used in `.travis.yml`
Add `COVERALLS_REPO_TOKEN` as is used in `docker-compose-travis.yml`